observability/src/test/java/io/grpc/observability/interceptors/LogHelperTest.java

```diff
@@ -68,7 +70,7 @@
 @RunWith(JUnit4.class)
 public class LogHelperTest {
 
-  private static final Charset US_ASCII = Charset.forName("US-ASCII");
+  private static final Charset US_ASCII = StandardCharsets.US_ASCII;
```

Comment:
Any reason you needed this additional variable `US_ASCII` ? You could directly use `StandardCharsets.US_ASCII` in all those places.

Reply:
Updated to use it inline.